dags/openshift_nightlies/util/manifest.py

```diff
@@ -1,69 +1,112 @@
 import yaml
-
+import sys
```

Comment:
Seems like these imports may be able to use some sorting. 
Maybe use Flake8 or Black linter ?

Reply:
yeah the imports need a cleanup. The main problem with using a linter atm is that the [PYTHONPATH is a bit messed up in Airflow](https://newbedev.com/apache-airflow-dag-cannot-import-local-module) to where I have to do some sys.path stuff to import local modules:

```python
import sys
from os.path import abspath, dirname
from os import environ

# need to call sys.path.insert before importing
sys.path.insert(0, dirname(dirname(abspath(dirname(__file__)))))
from util import var_loader, kubeconfig, constants
from tasks.index.status import StatusIndexer
from models.release import OpenshiftRelease
```

My VSCode Python extension will move the sys.path.insert call to after the local module imports everytime and it breaks the code. There's probably a better way to solve this though

Reply:
Its ok for the time being,  I can try and fix that as I start some contributions.